Mathlib/CategoryTheory/Bicategory/Functor/LocallyDiscrete.lean

```diff
@@ -0,0 +1,89 @@
+/-
+Copyright (c) 2024 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou
+-/
+import Mathlib.CategoryTheory.Bicategory.Functor.Pseudofunctor
+import Mathlib.CategoryTheory.Bicategory.LocallyDiscrete
+import Mathlib.CategoryTheory.Category.Cat
```

Comment:
```suggestion
import Mathlib.CategoryTheory.Bicategory.LocallyDiscrete
```
Although if you move the remaining functor code from `LocallyDiscrete` you probably have to import `Functor.Pseudofunctor` in the end anyways